kubernetes/kong-gw/is-config.yaml

```diff
@@ -3,8 +3,9 @@ kind: ConfigMap
 metadata:
   name: integration-service-api-config
 data:
-  PORT: "3000"
+  PORT_AUDIT: "3002"
```

Comment:
Port mapping must be managed by the runtime, not the environment variables.
These should be removed